modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryServiceImpl.java

```diff
@@ -243,6 +244,11 @@ public ObjectEntry getObjectEntry(long objectEntryId)
 		return objectEntry;
 	}
 
+	@Override
+	public List<ObjectEntry> getObjectEntries(long objectDefinitionId) {
+		return ObjectEntryUtil.findByObjectDefinitionId(objectDefinitionId);
```

Comment:
Is not usual call this type of **Util** class in this kind of class. Keep the pattern using **LocalService** here. If this finder doesn't not exists in the **LocalService**, expose it there (through the persistence finder).